File: cesar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace cesar
{
    internal class Program
    {
        static void Main(string[] args)
        {
		string alphabet = "abcdefghijklmnopqrstuvwxyz ";
		
		Console.Clear();
		System.Console.Write("input: ");
		string input = Console.ReadLine();
		System.Console.Write("offset: ");
		int offset = int.Parse(Console.ReadLine());
		Console.Clear();

		for(int i =0; i<input.Length;i++)
		{
			for(int j=0;j<alphabet.Length;j++)
			{
				if(input[i] == alphabet[j])
				{
					System.Console.Write(alphabet[(j+offset)%alphabet.Length]);
				}
			}
		}

		Console.ReadLine();
	             
        }
	

    }

}

File: cesar.cpp
#include <iostream>
#include <string>
int main(){
  
  std::string alpha = "abcdefghijklmnopqrstuvwxyz";

  std::cout << "Put in value: ";

  int change = 0;
  std::cin >> change;

  std::cout << std::endl << "input: ";
  std::string input;
  std::cin >> input;


  for(int i =0; i<input.length(); i++){
    
    for(int j = 0; j<alpha.length(); j++){
      if(input[i] == alpha[j]){
        std::cout <<alpha[i]<< ":" << alpha[((i+1)+change)%(alpha.length())] << "||||||" << "i + change: " << i << " | " << (i+j)%alpha.length() << std::endl;
    
      }
    }


  }

  
  
  

  
  


  return 0;
}
